src/js/pages/catalog/PackagesTab.js

```diff
@@ -197,10 +207,28 @@ class PackagesTab extends mixin(StoreMixin) {
     return <Trans id={copy} render={<Badge appearance={appearance} />} />;
   }
 
-  getCertifiedPackagesGrid(packages) {
-    if (this.state.searchString || packages.getItems().length === 0) {
+  getCertifiedPackagesGrid(packages, communityPackages) {
+    const isSearchActive = this.state.searchString !== "";
+    // If we are showing all packages, and we are filtering with a string
```

Comment:
This might be a little confusing at first, but I still think it's clear what we are trying to do.

Reply:
i pulled the functionality to show the "view all"-message out, so we don't complect rendering the `CertifiedPackagesGrid` with showing that message. they don't need to be coupled IMHO.